### ISSUE TITLE
Datahub-Rstudio package loading issues

### ISSUE THREAD

**Issue by seanmperez** [dlab-berkeley/R-Machine-Learning-Legacy#41]
When I open this workshop in datahub via the gitpuller, and run the first line:

```
renv::init()
```

I get the error message:

```
> knitr::opts_chunk$set(echo = TRUE)
Error in loadNamespace(x) : there is no package called ‘knitr’
```

When I try to install knitr manually, it takes a very long time.

**Comment by asteves** [comment]
Should now be solved with latest commit. Can you test again? @seanmperez

**Comment by seanmperez** [comment]
I get the same issue :( 

<img width="1420" alt="image" src="https://user-images.githubusercontent.com/75954522/156859652-5a15b211-6604-46f5-a416-067ddcb0b4ec.png">

**Comment by asteves** [comment]
@seanmperez What happens when you run line 15?

Furthermore, what happens if you run renv::restore()?

**Comment by seanmperez** [comment]
Hi Alex, apologies for the late reply. I think we are good at this point!

More details below:

1. When I run first click the datahub binder link, it opens up RStudio and clones the project in the desktop space.
2. When I open the project, the libraries needed are automatically attempted to install and I typically get a namespace error as shown above.
3. When I run `renv::restore()` I get a compilation issue (c++).
4. After, I try the `renv:init()` again, and after a while, it completes installation.
5. At this point, the notebook opens up properly when connected to the project.

Long story short, it can work but is a bit finicky on datahub! I think we are all good considering the line in the installation instructions below:

# Is R not working on your laptop?
This workshop makes use of many packages within the R ecosystem. For that reason, we have found it to be easier to use R on your local machine.
